<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>con2stm</class>
 <widget class="QWidget" name="con2stm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>TCP:IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="con_eidt_tcpip">
           <property name="text">
            <string>192.168.1.192</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TCP:port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="con_eidt_tcpport">
           <property name="text">
            <string>5000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="con_2tcp">
         <property name="text">
          <string>CON</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tcp_sendtest">
         <property name="text">
          <string>下载界面</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="con2stmclose">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>UDP:IP</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="con_edit_udpip"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>UDP:port</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="con_edit_udpport"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="con_2udp">
         <property name="text">
          <string>CON</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>USART:baud115200</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="con_usartcombox"/>
       </item>
       <item>
        <widget class="QPushButton" name="con_2usart">
         <property name="text">
          <string>CON</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>data_send:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="con_sendtextBrowser"/>
         </item>
         <item>
          <widget class="QPushButton" name="con_clearsend">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>data_get:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="con_gettextBrowser"/>
         </item>
         <item>
          <widget class="QPushButton" name="con_clearget">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
